app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -1,12 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import cityImage from '@images/buildings.png';
+import pvdImage from '@images/buildings.png';
+import bldImage from '@images/boulder_buildings.svg';
 import { yValue, xValue } from '@lib/circle-math';
 import { timeDiffInMinutes, timeAndDateForTimezone } from '@lib/datetime';
 import { colors } from '@lib/theme';
 import styled, { keyframes } from 'styled-components';
 import MoonPhase from '@weather/moon-phase';
 
+const getCityImage = cityName => {
+  if (cityName === 'Boulder') return bldImage;
+  return pvdImage;
+};
+
+const PvdStyle = styled.circle`
+offset-path: path('${p => p.arcPath}');
+animation: ${p => p.arcAnimation} 1000ms 1 normal forwards ease-in-out;
+r: ${p => p.ballRadius};
+fill: ${colors.pink}`;
+
+const BldStyle = styled.circle`
+offset-path: path('${p => p.arcPath}');
+animation: ${p => p.arcAnimation} 1000ms 1 normal forwards ease-in-out;
+r: ${p => p.ballRadius};
+fill: ${colors.yellow2}`;
```

Comment:
Nit: you could still have one component and pass in the fill color as a property.

Reply:
That makes a lot of sense, this should be fixed now!